values.yaml

```diff
@@ -120,4 +106,29 @@ relay:
 
   ## Secrets don't like '-'
   secrets:
-    relayCogToken: 
+    relayCogToken:
+
+nginx:
+  name: nginx
+  replicaCount: 1
+  image:
+    repository: nginx
+    tag: 1.11
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  serviceType: LoadBalancer
+  ingress:
```

Comment:
Fair point, though in terms of larger cluster management having random isolated ingress points that don't bubble up to your `Ingress` resources (if you are using them) would get undesirable at any sort of scale. I'm alright with leaving it as is.

As an example for me to use this chart I would be changing the nginx `serviceType` to `ClusterIP` and the `Ingress` resource would be directing traffic from the load balancer that's routing ALL traffic into my cluster. This seems like a forced GKE limitation.